modular_skyrat/modules/customization/modules/clothing/suits/misc.dm

```diff
@@ -287,28 +287,19 @@
 	mutant_variants = NONE
 	armor = list(MELEE = 10, BULLET = 10, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
 
-/obj/item/clothing/suit/corgisuit/en
-	name = "\improper super-hero E-N suit"
+/obj/item/clothing/suit/ashwalkermantle
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
-	icon_state = "ensuit"
+	name = "tanned hide"
+	desc = "The tanned hide of some brown furred creature."
+	icon_state = "mantle_liz"
+	mutant_variants = NONE
+
+/obj/item/clothing/suit/ashwalkermantle/cape
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	name = "brown leather cape"
+	desc = "An ash coated cloak."
+	icon_state = "desertcloak"
 	mutant_variants = NONE
 
-/obj/item/clothing/suit/corgisuit/en/New()
```

Comment:
Unrelated changes here

Reply:
I have no idea what's going on there tbh

Reply:
More specifically not sure how to fix it. Amelia said it'd be fine